src/network/protocol/messages/notfound.zig

```diff
@@ -0,0 +1,131 @@
+const std = @import("std");
+const protocol = @import("../lib.zig");
+const Sha256 = std.crypto.hash.sha2.Sha256;
+
+/// NotFoundMessage represents the "notfound" message
+///
+/// https://developer.bitcoin.org/reference/p2p_networking.html#notfound
+pub const NotFoundMessage = struct {
+    inventory: []const InventoryVector,
+
+    pub const InventoryVector = struct {
+        type: u32,
+        hash: [32]u8,
+    };
```

Comment:
This struct will be common to different messages. Move it under /messages/lib.zi

Reply:
Okay boss

Reply:
@tdelabro I have moved it